scripts/analyze_failed_jobs/action_plugins/parsejobs.py

```diff
@@ -0,0 +1,40 @@
+import glob, pickle, buildbot, re, sys, cStringIO as StringIO
```

Comment:
Even if it an ansible module, it should have a docstring describing what it is, and what it is used for.